### PR TITLE
Marp 6182 fix tracing path variable parameters

### PR DESCRIPTION
**Pull request by ibu04**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+clean:
+	mvn clean
+
+package: clean
+	mvn -B package -DskipTests=true
+
+deploy: package
+	mvn -B deploy -DskipTests=true
```
Comment (denis-tazhkenov) [inline Makefile:9]: Don't we need separate steps for snapshot and release deployment? How would it know to which repository to deploy?

File: README.adoc [hunk 2]
```diff
@@ -712,4 +714,136 @@ added after the original pull request but before a merge.
   other target branch in the main project).
 * When writing a commit message please follow https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html[these conventions],
   if you are fixing an existing issue please add `Fixes gh-XXXX` at the end of the commit
-  message (where XXXX is the issue number).
\ No newline at end of file
+  message (where XXXX is the issue number).
```
Comment (denis-tazhkenov) [inline README.adoc]: Where are these changes from? Looks like something from a newer version.

File: benchmarks/pom.xml [hunk 3]
```diff
@@ -17,7 +17,7 @@
 	<name>Benchmarks</name>
 	<description>Benchmarks (JMH)</description>
 	<groupId>org.springframework.cloud</groupId>
-	<version>1.3.7.BUILD-SNAPSHOT</version>
+	<version>1.3.6.CAREEM1-SNAPSHOT</version>
```
Comment (denis-tazhkenov) [inline benchmarks/pom.xml]:
```suggestion
	<version>1.3.6.CAREEM2-SNAPSHOT</version>
```

Comment (denis-tazhkenov) [reply benchmarks/pom.xml]: And the same in the other places

File: ops/cd.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+stages:
+  - deploy-to-artifactory
+
+
+jobs:
+  - name: deploy-to-artifactory
+    stage: deploy-to-artifactory
+    profile: qa
+    cmd: make deploy
+    branch: 1.3.x.custom
```
Comment (denis-tazhkenov) [inline ops/cd.yml]: please don't forget newlines at the end of files

File: ops/ci.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+team: mobility-be-matching
+email: mobility_be_matching@careem.com
+slack: mobility-be-matching-notifications
+project: maven
+agent: pipeline-generator-agent
+source: bitbucket
+branch: master
```
Comment (denis-tazhkenov) [inline ops/ci.yml]: here too

**Review by denis-tazhkenov — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (ibu04) [reply README.adoc]: these might have changed after cloning the newer version of forked repo, do you want me to revert these back?

File: Makefile (hunk 1 above)
Comment (ibu04) [reply Makefile:9]: > If your project.version contains SNAPSHOT (f.e., 1.2-SNAPSHOT) and you execute mvn deploy, artifacts will be deployed to your snapshot repository. If it doesn't (f.e., 1.2) - they will be deployed to your release repository.

https://stackoverflow.com/questions/9461225/how-to-differentiate-between-deploying-releases-vs-snapshots

File: README.adoc (hunk 2 above)
Comment (denis-tazhkenov) [reply README.adoc]: yes

**Comment by denis-tazhkenov** [comment]
I'm trying to test this version, but it's not available in Artifactory?
Upd. Now works. Tested, confirmed.

**Review by denis-tazhkenov — APPROVED** [review]

**Pull request merged**